.travis.yml

```diff
@@ -1,8 +1,20 @@
-language: node_js
 sudo: false
+
+git:
+  depth: 1
+
+branches:
+  only:
+    - master
```

Comment:
Previously each PR would trigger 2 Travis CI jobs, now only the required Travis CI PR job testing the merge into `master` is ran.